### DISCUSSION TITLE
Modifiers

### DISCUSSION THREAD

**Discussion by JaSimps5** [CMSgov/hospital-price-transparency#148] [Q&A] [answered]
I have two questions regarding modifiers since they seem to be treated slightly different than other codes. 
1. If having a modifier present means the item doesn't get reimbursed by the payor, but 0's are not allowed, is it acceptable that we leave the percentage blank and only populate the additional notes for that negotiated contract?
2. The presence of some modifiers is not a straight percentage, but a lesser of calculation. I want to verify if we include that calculation in the algorithm field?

**Comment by RidenourK** [comment, marked as answer]
@ JaSimps5. Thank you for your questions.  

Please refer to these conditional requirements in the [CMS Hospital Price Transparency Data Dictionary GitHub](https://github.com/CMSgov/hospital-price-transparency/tree/master/documentation) site which indicates if a modifier is encoded without an item or service, then a “Description” and one of the following is the minimum information required: “Payer-specific Negotiated Charge: Dollar Amount”, “Payer-specific Negotiated Charge: Percentage”, “Payer-specific Negotiated Charge: Algorithm”, “Additional Generic Notes”, or “Additional Payer-Specific Notes”. Required beginning 1/1/2025.
